taxi/views.py

```diff
@@ -44,6 +63,23 @@
     model = Car
 
 
+class CarCreateView(CreateView):
+    model = Car
+    form_class = CarForm
+    success_url = reverse_lazy("car-list")
+
+
+class CarUpdateView(UpdateView):
+    model = Car
+    form_class = CarForm
+    success_url = reverse_lazy("car-list")
+
+
+class CarDeleteView(DeleteView):
```

Comment:
Consider adding `LoginRequiredMixin` to `CarDeleteView` to ensure that only authenticated users can delete a car.